frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunLogs.tsx

```diff
@@ -92,11 +92,11 @@ class PipelineRunLogs extends React.Component<PipelineRunLogsProps, PipelineRunL
         : undefined;
     const resources = taskCount > 0 && [
       {
-        name: _.get(taskRunFromYaml[activeItem], ['status', 'podName'], ''),
+        selector: { 'tekton.dev/taskRun': activeItem },
         kind: 'Pod',
         namespace: obj.metadata.namespace,
         prop: `obj`,
-        isList: false,
+        isList: true,
```

Comment:
any reason for converting it to an array.

Reply:
Earlier we used to query by Pod name, which would return a single Pod. But now we are searching Pod by label which would return an array.